src/index.html

```diff
@@ -8,9 +8,43 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1 class="title">Search bar airbnb</h1> -->
+    <form 
+      action="#" 
+      method="GET" 
+      data-qa="big" 
+    >
+      <div class="big_Field">
+        <div class="text-Field_icon">
+          <input 
+            data-qa="keypress"
+            type="text" 
+            class="firstBar"
```

Comment:
Here you use camel case and below you use snake case. Try to write in one style.